pandas-stubs/io/json/_json.pyi

```diff
@@ -42,16 +42,42 @@ def read_json(
         Literal["strict", "ignore", "replace", "backslashreplace", "surrogateescape"]
         | None
     ) = ...,
+    lines: bool,
```

Comment:
Should be `lines: Literal[True]`, because you can only specify `chunksize` when `lines == True`